src/app/core/guideline.service.ts

```diff
@@ -70,7 +70,7 @@ export class GuidelineService {
     return {
       text: filter.text || filter.filterText,
       year: filter.year !== '' ? filter.year : undefined,
-      levels: filter.levels !== [] ? filter.levels : undefined,
+      levels: filter.levels.length > 0 ? filter.levels : undefined,
```

Comment:
Just curious, why is this a part of your PR?

Reply:
This line was throwing an error that prevented me from running production